core/src/components/datetime/datetime.tsx

```diff
@@ -341,6 +341,10 @@ export class Datetime implements ComponentInterface {
    * dates are selected. Only used if there are 0 or more than 1
    * selected (i.e. unused for exactly 1). By default, the header
    * text is set to "numberOfDates days".
+   *
+   * Developers who wish to access "this" inside of the function
```

Comment:
```suggestion
   * Angular developers who wish to access "this" inside of the function
```
Should it specify the framework? My concern is that other framework users might get confused with the comment.